tests/page_test.go

```diff
@@ -174,6 +174,84 @@ func TestPageEvaluate(t *testing.T) {
 	})
 }
 
+func TestPageEvaluateMapping(t *testing.T) {
+	tests := []struct {
+		name    string
+		script  string
+		want    any
+		wantErr string
+	}{
+		{
+			name:   "()_=>_0",
```

Comment:
Can we give these names descriptive names like `arrow_func`, `func_null_return`, `async_func`, `not_a_func`, etc.? It'll be harder to run them this way on the command-line and they don't tell us the goal of the test cases.

Reply:
Amended in 972a8da8ade8e5e9cf667cc5ee865ab2e7099f5e